{"abstract":[{"text":"Indicates that a server is terminating the connection because","type":"text"},{"text":" ","type":"text"},{"text":"it encountered an unexpected condition that prevented it from","type":"text"},{"text":" ","type":"text"},{"text":"fulfilling the request.","type":"text"},{"text":" ","type":"text"},{"text":"Corresponds to code 1011.","type":"text"}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode"]]},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unexpectedServerError"}],"platforms":["Linux"],"languages":["swift"]}],"kind":"declarations"}],"identifier":{"url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode\/unexpectedServerError","interfaceLanguage":"swift"},"variants":[{"paths":["\/documentation\/niowebsocket\/websocketerrorcode\/unexpectedservererror"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"metadata":{"title":"WebSocketErrorCode.unexpectedServerError","symbolKind":"case","roleHeading":"Case","role":"symbol","modules":[{"name":"NIOWebSocket"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unexpectedServerError","kind":"identifier"}],"externalID":"s:12NIOWebSocket03WebB9ErrorCodeO016unexpectedServerD0yA2CmF"},"schemaVersion":{"minor":3,"major":0,"patch":0},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOWebSocket":{"role":"collection","kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket","abstract":[],"type":"topic","url":"\/documentation\/niowebsocket","title":"NIOWebSocket"},"doc://co.pointfree.VaporRouting/documentation/NIOWebSocket/WebSocketErrorCode":{"kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode","role":"symbol","url":"\/documentation\/niowebsocket\/websocketerrorcode","abstract":[{"type":"text","text":"An enum that represents websocket error codes."}],"type":"topic","title":"WebSocketErrorCode","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"WebSocketErrorCode"}],"navigatorTitle":[{"kind":"identifier","text":"WebSocketErrorCode"}]},"doc://co.pointfree.VaporRouting/documentation/NIOWebSocket/WebSocketErrorCode/unexpectedServerError":{"abstract":[{"text":"Indicates that a server is terminating the connection because","type":"text"},{"text":" ","type":"text"},{"text":"it encountered an unexpected condition that prevented it from","type":"text"},{"text":" ","type":"text"},{"text":"fulfilling the request.","type":"text"},{"text":" ","type":"text"},{"text":"Corresponds to code 1011.","type":"text"}],"kind":"symbol","title":"WebSocketErrorCode.unexpectedServerError","type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode\/unexpectedServerError","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unexpectedServerError","kind":"identifier"}],"url":"\/documentation\/niowebsocket\/websocketerrorcode\/unexpectedservererror","role":"symbol"}}}